webpack/assets/javascripts/react_app/components/common/forms/TextInput.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const TextInput = (props) => {
+  return (
+    <div className="clearfix">
```

Comment:
since the div structure more or less repeat it self, does it make sense to have a common base component with all of that structure once vs in each component? for example:

```jsx
return (
<CommonForm>
  <input type...>
</CommonForm>
)
```